gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -16,7 +16,13 @@ security:
 
 servers:
     - url: /management/v2
-      description: Gravitee.io APIM - Management API -v2
+      description: Gravitee.io APIM - Management API - v2
+    - url: /management/v2/organizations/{orgId}
```

Comment:
can't we add a third one ?

`- url: /management/v2/organizations/{orgId}/environments/{envId}`

and remove the 'environnments' part of the paths ?